soc/riscv/riscv-privilege/virt/soc.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2020 Cobham Gaisler AB
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __RISCV_VIRT_SOC_H_
+#define __RISCV_VIRT_SOC_H_
+
+#include <soc_common.h>
+#include <devicetree.h>
+
+#define RISCV_MTIME_BASE             0x0200BFF8
+#define RISCV_MTIMECMP_BASE          0x02004000
+
```

Comment:
It's better to sort by address?

Reply:
The reason for the current layout is to keep the file consistent with the other `soc.h` files in `soc/riscv/riscv-privilege`. So I think we should leave it as is to make `diff`ing between the files smooth.

Reply:
Thanks! I see.